### DISCUSSION TITLE
How to install an older version

### DISCUSSION THREAD

**Discussion by roger-net** [AzuraCast/AzuraCast#3873] [Q&A]
What is the command to install an older version of Azuracast on a clean vps?

**Comment by Madpeterz** [comment]
Are you going to be using docker and the auto installer script?

**Comment by roger-net** [comment]
> Are you going to be using docker and the auto installer script?

Yes, I will use the docker method.

**Comment by roger-net** [comment]
I would like to install version 0.10.2 on a clean vps. How do I do that?

**Comment by YosefAdPro** [comment]
Try this
./docker.sh install 0.10.2

**Comment by roger-net** [comment]
> Try this
> ./docker.sh install 0.10.2

I've tried this command and it didn't work

**Comment by BusterNeece** [comment]
There is currently no built-in way to install an older version of AzuraCast. We prefer that all users be running the latest version, especially for security reasons as we update our software dependencies and fix bugs.

If you need to test with an older version, you can download the release file for it from the GItHub Releases page; at that point, only the Ansible installation method will work (as the Docker installation will simply pull the latest images down instead of using local files, unless you follow the Local Development instructions).
